enrich-points-with-boundaries/geodatabase/src/test/scala/com/precisely/bigdata/sample/spark/VerificationDriver.scala

```diff
@@ -21,6 +21,15 @@ object VerificationDriver {
     val sparkConf = new SparkConf()
     sparkConf.setIfMissing("spark.master", "local[*]")
     sparkConf.set("spark.sql.autoBroadcastJoinThreshold", "-1")
+
+    var sparkMajorVersion=org.apache.spark.SPARK_VERSION.split('=')(0).split('.')(0).toInt
+    if(sparkMajorVersion >= 3){
+      var sparkMinorVersion=org.apache.spark.SPARK_VERSION.split('=')(0).split('.')(1).toInt
```

Comment:
minor change, I think you should use "val" instead of "var" for both sparkMajor\MinorVersion as these should be immutable.

Reply:
I incorporate the advised changes in the latest build.